samples/algorithms/integer-factorization/README.md

```diff
@@ -11,3 +11,27 @@ a part of Microsoft.Quantum.Canon library.
 - [Program.cs](./Program.cs): C# console application running Shor's algorithm
   on Quantum simulator
 - [IntegerFactorization.csproj](./IntegerFactorization.csproj): Main C# project for the sample.
+
+# Flame Graph Visualization #
+
+This sample also contains an adapter for the [ResourcesEstimator](https://github.com/microsoft/qsharp-runtime/tree/974a385cc57c2b663e8134c1f3170f9cb8ae5fb1/src/Simulation/Simulators/ResourcesEstimator) allowing it to produce a resource utilization stack trace which can then be used to produce a flame graph. More details can be found in [this article](https://aman3014.medium.com/flame-graphs-for-q-f4f9bb076d88).
+
+To generate a flame graph, follow these steps:
+
+1. Clone the [flamegraph repository](https://github.com/brendangregg/FlameGraph)
+2. Install perl
```

Comment:
```suggestion
2. Install Perl
```
May be out of scope, but even a pointer as to how to do so may be helpful. In particular, I'd think that Perl is likely to come by default on most Linux and macOS systems, such that using chocolatey to install StrawberryPerl may be useful for Windows users.

Have you checked if this all works in the samples devcontainer for VS Code / GH Codespaces / mybinder.org usage?